pkg/share/manager/jsoncs3/jsoncs3.go

```diff
@@ -511,8 +511,6 @@ func (m *Manager) UpdateShare(ctx context.Context, ref *collaboration.ShareRefer
 				toUpdate.Permissions = updated.Permissions
 			case "expiration":
 				toUpdate.Expiration = updated.Expiration
-			case "hide":
-				toUpdate.Hide = updated.Hide
```

Comment:
Hmm not directly, since this functionality was moved to `UpdateReceivedShare`.... oh wait... the testsuite is probably still hammering the UpdateShare function....

Reply:
And there was an obsolete field left in the Fieldmask... my bad....